pkg/controller/kogitoruntime/deployer.go

```diff
@@ -164,7 +201,19 @@ func applyProtoBufConfigurations(deployment *v1.Deployment, kogitoService v1alph
 					Name:      downwardAPIVolumeName,
 					MountPath: downwardAPIVolumeMount,
 				})
-		deployment.Spec.Template.Spec.Containers[0].Lifecycle =
-			&corev1.Lifecycle{PostStart: &corev1.Handler{Exec: &corev1.ExecAction{Command: podStartExecCommand}}}
 	}
 }
+
+func getHTTPFileBytes(fileURL string) ([]byte, error) {
```

Comment:
can't instead return the bytes return the json list already?

Reply:
I think this is not a JSON, but the protobuf file. @Kevin-Mok can you confirm? If so, would be better to rethink the function name to give context.

Reply:
I'm using this function to get both the list and protobufs. What function can I use to get the JSON list directly?

Reply:
I believe that we can keep this name, @Kevin-Mok. @spolti wdyt?

Reply:
@ricardozanini sound good to me.